docs/install.md

```diff
@@ -5,78 +5,99 @@ feature_maturity: stable
 enterprise: 'no'
 ---
 
-Spark is available in the Universe and can be installed by using either the web interface or the DC/OS CLI.
+Spark is available in the Universe and can be installed by using either the GUI or the DC/OS CLI.
```

Comment:
For DC/OS docs, it should be `GUI`. This has not been globally fixed, yet, but is the goal.